precompile/contracts/hubblebibliophile/clearing_house.go

```diff
@@ -95,3 +94,11 @@ func GetMaintenanceMargin(stateDB contract.StateDB) *big.Int {
 func GetMinAllowableMargin(stateDB contract.StateDB) *big.Int {
 	return new(big.Int).SetBytes(stateDB.GetState(common.HexToAddress(CLEARING_HOUSE_GENESIS_ADDRESS), common.BytesToHash(common.LeftPadBytes(big.NewInt(MIN_ALLOWABLE_MARGIN_SLOT).Bytes(), 32))).Bytes())
 }
+
+func GetPositionSizes(stateDB contract.StateDB, trader *common.Address) []*big.Int {
```

Comment:
if we are not using this in subnet-evm side we can make this private function.

Reply:
done